flink-end-to-end-tests/flink-quickstart-test/pom.xml

```diff
@@ -20,27 +20,43 @@ under the License.
 <project xmlns="http://maven.apache.org/POM/4.0.0"
 		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-
-	<modelVersion>4.0.0</modelVersion>
-
 	<parent>
 		<groupId>org.apache.flink</groupId>
 		<artifactId>flink-end-to-end-tests</artifactId>
 		<version>1.6-SNAPSHOT</version>
-		<relativePath>..</relativePath>
 	</parent>
+	<modelVersion>4.0.0</modelVersion>
 
-	<artifactId>flink-elasticsearch5-test</artifactId>
-	<name>flink-elasticsearch5-test</name>
+	<artifactId>flink-quickstart-test</artifactId>
+	<name>flink-quickstart-test</name>
 	<packaging>jar</packaging>
 
+	<properties>
+		<scala.binary.version>2.11</scala.binary.version>
+	</properties>
+
 	<dependencies>
 		<dependency>
 			<groupId>org.apache.flink</groupId>
 			<artifactId>flink-streaming-java_${scala.binary.version}</artifactId>
 			<version>${project.version}</version>
 			<scope>provided</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-streaming-scala_${scala.binary.version}</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-connector-elasticsearch_${scala.binary.version}</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-connector-elasticsearch2_${scala.binary.version}</artifactId>
```

Comment:
depending multiple connector versions in one modules is a recipe for dependency conflicts and should be avoided at all costs. There's a reason why the existing examples and connector versions reside in different modules.

Reply:
@zentol removed multiple connector versions